LEGO1/mxomni.cpp

```diff
@@ -278,18 +278,23 @@ MxResult MxOmni::Create(MxOmniCreateParam& p)
 // OFFSET: LEGO1 0x100afe90
 void MxOmni::Destroy()
 {
-	// FIXME: Stub
+	{
```

Comment:
fix indentation

Reply:
This is now governed by `clang,-format`. If the action succeeds, nothing need be changed